### ISSUE TITLE
What's the numpy projection function?

### ISSUE THREAD

**Issue by ijnek** [ros-sports/ipm#21]
https://github.com/ros-sports/ipm/blob/017922a9ddc404b8f5ec6ae19c46035553544332/ipm_library/ipm_library/ipm.py#L76

What's the numpy projection function?

**Comment by Flova** [comment]
We have the projection function for Point2DStamped msgs and the projection of numpy arrays containing multiple pixels. The first one is an wrapper of the second one. We could also say point cloud projection instead of numpy projection, but the input is a numpy array not a PointCloud msg.

**Comment by ijnek** [comment]
Ah got it, I might reword it to

``# Convert point to numpy array and utilize map_points()`` in #24.
